src/elements/context-menu/context-menu-item.svelte

```diff
@@ -0,0 +1,53 @@
+<svelte:options immutable tag="v-context-menu-item" />
+
+<script lang="ts">
+import cx from 'classnames';
+import { addStyles } from '../../lib/index';
+import { dispatcher } from '../../lib/dispatch';
+import { htmlToBoolean } from '../../lib/boolean';
+
+const dispatch = dispatcher();
+
+addStyles();
+
+export let icon: string;
+export let divider: string;
+export let variant: 'primary' | 'danger' = 'primary';
+export let label: string;
+
+$: isDivider = htmlToBoolean(divider, 'divider');
+
+const handleSelect = () => {
+  dispatch('select', { value: label });
+};
+</script>
+
+<div>
```

Comment:
Should have a `role` of [`menuitem`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menuitem_role)